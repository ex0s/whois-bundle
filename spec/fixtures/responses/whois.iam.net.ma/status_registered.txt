%rwhois V-1.5:003fff:00 whois.iam.net.ma (by Network Solutions, Inc. V-1.5.9.5)
domain:Class-Name:domain
domain:ID:333.google.ma
domain:Auth-Area:google.ma
domain:Guardian;I:444.google.ma
domain:Domain-Name:google.ma
domain:Primary-Server;I:111.google.ma
domain:Secondary-Server;I:112.google.ma
domain:Organization;I:777.google.ma
domain:Owner-Contact;I:221.google.ma
domain:Admin-Contact;I:222.google.ma
domain:Tech-Contact;I:223.google.ma
domain:Billing-Contact;I:222.google.ma
domain:Created:20090324
domain:Updated:20090324
domain:Updated-By:dnsmaster@iam.net.ma

%ok
