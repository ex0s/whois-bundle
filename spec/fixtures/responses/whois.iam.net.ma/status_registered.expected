#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2009-03-24}

#updated_on
  %s %CLASS{time}
  %s %TIME{2009-03-24}

#expires_on
  %s %ERROR{AttributeNotSupported}
